assets/stylesheets/bookingsync/_form.scss

```diff
@@ -64,6 +64,27 @@
     transition-timing-function: cubic-bezier(.4,0,.2,1);
   }
 
+  &.has-icon {
+    i {
+      position: absolute;
+      bottom: 15px;
+      color: $gray-light;
+      &.icon-input {
```

Comment:
minor styling detail, but good to keep an empty line between each block (new definition). It makes it much easier to read.